AttributeError: module 'spinalcordtoolbox' has no attribute 'deepseg'

### Error

```
=================================== FAILURES ===================================
______________________________ test_install_model ______________________________
[gw0] linux -- Python 3.6.12 /repo/python/envs/venv_sct/bin/python
    def test_install_model():
        """
        Download all models, to allow subsequent tests on the model packages.
        :return:
        """
>       for name_model, value in sct.deepseg.models.MODELS.items():
E       AttributeError: module 'spinalcordtoolbox' has no attribute 'deepseg'
unit_testing/test_deepseg.py:31: AttributeError
_______________________________ test_model_dict ________________________________
[gw0] linux -- Python 3.6.12 /repo/python/envs/venv_sct/bin/python
    def test_model_dict():
        """
        Make sure all fields are present in each model.
        :return:
        """
>       for key, value in sct.deepseg.models.MODELS.items():
E       AttributeError: module 'spinalcordtoolbox' has no attribute 'deepseg'
unit_testing/test_deepseg.py:43: AttributeError
_________________________ test_segment_nifti[params0] __________________________
[gw0] linux -- Python 3.6.12 /repo/python/envs/venv_sct/bin/python
params = {'fname_image': 'sct_testing_data/t2s/t2s.nii.gz', 'fname_seg_manual': 'sct_testing_data/t2s/t2s_seg-deepseg.nii.gz', 'model': '/repo/data/deepseg_models/t2star_sc'}
    @pytest.mark.parametrize('params', param_deepseg)
    def test_segment_nifti(params):
        """
        Uses the locally-installed sct_testing_data
        """
        fname_out = 't2s_seg_deepseg.nii.gz'
>       output = sct.deepseg.core.segment_nifti(
            params['fname_image'], params['model'], param={'o': fname_out})
E       AttributeError: module 'spinalcordtoolbox' has no attribute 'deepseg'
```

### Cause

Here is what I believe happened:

* #2997 removed imports from subpackage `__init__.py` files. As a consequence, scripts are required to explicitly import subpackage modules.
* #2996 removed subpackage module imports from `test_deepseg`. (`import spinalcordtoolbox.deepseg.core`, `import spinalcordtoolbox.deepseg.models`)

These were merged at roughly the same time, so the errors slipped through without thorough testing.

--- a/unit_testing/test_deepseg.py
+++ b/unit_testing/test_deepseg.py
@@ -11,6 +11,8 @@ import nibabel
 
 import spinalcordtoolbox as sct
 from spinalcordtoolbox.utils import sct_test_path
+import spinalcordtoolbox.deepseg.core
+import spinalcordtoolbox.deepseg.models
 
 
 param_deepseg = [

